### ISSUE TITLE
Is there any chance to use this extension on Zotero?/有没有可能在Zotero上使用本插件？

### ISSUE THREAD

**Issue by zzeitt** [gdh1995/vimium-c#719]
**What command or commands**

Install this extension on Zotero.

**How should a feature do**

When a user has done such steps:

1. Installed the zotero-version VimiumC on Zotero.

then Vimium C should do:

enabling the vim-like keybinding for zotero.

**Browser and OS**

- Browser name: Zotero 6.0.14
- OS name: Win 10

**Comment by gdh1995** [comment]
No. It seems Zotero doesn't support web-extensions - on https://www.zotero.org/support/plugins#developer_tools its plugins only use the old Firefox Plugin APIs. So Vimium C can not run on it.

But maybe you can try installing https://github.com/akhodakivskiy/VimFx on it:
![image](https://user-images.githubusercontent.com/5547703/188262118-826b8520-87bd-46e1-a528-6dfc46331cb4.png)
and 
![image](https://user-images.githubusercontent.com/5547703/188262135-afda15b2-161c-4479-9c5a-8c3c0a99af04.png)

**Comment by zzeitt** [comment]
感谢你的提议，我试了一下，安装不上，提示版本不兼容。

![image](https://user-images.githubusercontent.com/30381410/188264437-dcbacc11-508c-4b20-b9b4-69d375c5a555.png)

出现这条提示的原因可参考这里(https://groups.google.com/g/zotero-dev/c/_LuAHXs1Q8A/m/zg4o9wroAgAJ) 。

但是令我感到奇怪的是另一个插件saka-key(https://addons.mozilla.org/en-US/firefox/addon/saka-key/) 能够成功安装，但是无法在zotero中使用。

由于我对web插件开发不了解，所以请教您一下出现这种情况的原因是什么？
